Gemfile

```diff
@@ -10,7 +10,7 @@ def vanagon_location_for(place)
   end
 end
 
-gem 'vanagon', *vanagon_location_for(ENV['VANAGON_LOCATION'] || '~> 0.5.8')
+gem 'vanagon', *vanagon_location_for(ENV['VANAGON_LOCATION'] || '~> 0.5.7')
```

Comment:
If you wanted to pin to `0.5.7`, I think you should have removed the `~>`

I have removed my `.bundle`, and `Gemfile.lock`, but now trying to bundle install locally for me now produces this error:

```
Fetching git://github.com/puppetlabs/packaging.git
Fetching gem metadata from https://rubygems.org/...........
Fetching version metadata from https://rubygems.org/..
Resolving dependencies...
Installing rake 11.1.2
Installing ast 2.2.0
Installing parser 2.3.0.7
Installing astrolabe 1.3.1
Installing json 1.8.3 with native extensions
Installing redis 3.2.2
Installing lock_manager 0.1.1
Using packaging 0.4.3.343.g4709767 from git://github.com/puppetlabs/packaging.git (at master)
Installing powerpack 0.1.1
Installing rainbow 2.1.0
Installing ruby-progressbar 1.7.5
Installing rubocop 0.34.2

Gem::RemoteFetcher::FetchError: bad response Not Found 404 (https://rubygems.global.ssl.fastly.net/gems/vanagon-0.5.8.gem)
An error occurred while installing vanagon (0.5.8), and Bundler cannot continue.
Make sure that `gem install vanagon -v '0.5.8'` succeeds before bundling.
```